### PR TITLE
update qps and burst for webhook

### PR DESCRIPTION
**Pull request by ldpliu**
Signed-off-by: ldpliu <daliu@redhat.com>

### CODE REVIEW

**Review by qiujian16 — COMMENTED** [review]
maybe we should put this as flag and set some default value

**Comment by ldpliu** [comment]
> maybe we should put this as flag and set some default value

it's not easy to add the qps and burst flag for webhook. 
As current webhook use the `o.RecommendedOptions`
 https://github.com/open-cluster-management-io/registration/blob/fd33e20727c4d295f49b18c249d279369426c230/pkg/cmd/webhook/webhook.go#L46

**Comment by qiujian16** [comment]
I think you can create another AddFlags options for this, right?

**Comment by ldpliu** [comment]
> I think you can create another AddFlags options for this, right?

Done, please help to  review again.

File: pkg/cmd/webhook/webhook.go [hunk 1]
```diff
@@ -44,6 +46,42 @@ func NewAdmissionHook() *cobra.Command {
...
+	if err != nil {
+		return err
+	}
+	return server.GenericAPIServer.PrepareRun().Run(stopCh)
+}
+
+// Config contains the server (the webhook) cert and key.
+type Options struct {
+	QPS   float32
+	Burst int
```
Comment (qiujian16) [inline pkg/cmd/webhook/webhook.go]: I would put adminssonServerOption in this Option. So all addFlags can be done in options.AddFlat.

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

this is also needed for work

**Pull request merged**